### DISCUSSION TITLE
Running Kanachan without Docker

### DISCUSSION THREAD

**Discussion by adsf0427** [Cryolite/kanachan#72] [Q&A]
Hello, I'm interested in using this repository on a compute node where I don't have Docker or sudo access. I'd appreciate some guidance on how to run the code directly without relying on the Docker setup.

### Current Setup
Currently, the repository uses Docker as its primary running script. While this is great for ensuring consistency across different environments, it poses challenges in environments where Docker isn't available or allowed.

### Request
Could the author or other contributors provide instructions on:

1. Setting up the necessary environment without Docker?
2. Installing dependencies directly (preferably without requiring sudo)?
3. Any modifications needed to run scripts or commands that are currently in the Dockerfile?

### Use Case
I'm planning to use this on a compute cluster for training purposes. Any specific considerations for high-performance environments would be especially helpful.
Thank you in advance for any assistance or insights you can provide!

**Comment by suiniara** [comment]
Fortunately, there are options available for setting up the necessary environment without Docker:
Local Installation Script:In the 'develop' branch of the repository, there is a script specifically designed for local installation. You can find it at:https://github.com/Cryolite/kanachan/blob/develop/bin/install.sh This script should help you set up the environment and install dependencies without requiring Docker or sudo access.
Wiki Documentation:The project's wiki also addresses this issue. You can find detailed information about local installation at:https://github.com/Cryolite/kanachan/wiki/Local-Install This wiki page likely provides step-by-step instructions for setting up the environment, installing dependencies, and any necessary modifications to run the scripts without Docker.
> While there may be some issues with the above script, you could try comparing it to kanachan dockerfile
